DirectEmployers/MyJobs
# [PD-4023] remove references that caused an error in search

## What's this PR do?
Removes references to outreachEmail, which we removed from the NUO records

## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
The issue that brought this to attention was that the search stopped working in NUO.
Ensure that works.

The rest of the changes only involved automated tests, but clicking through the app never hurts.

## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
